src/libpgmoneta/utils.c

```diff
@@ -1496,6 +1496,26 @@ pgmoneta_copy_postgresql(char* from, char* to, char* base, char* server, char* i
    char* to_buffer = NULL;
    struct dirent* entry;
    struct stat statbuf;
+   char** restore_last_files_names = NULL;
+
+   if (pgmoneta_get_restore_last_files_names(&restore_last_files_names))
+   {
+      goto error;
+   }
+
+   for (int i = 0; restore_last_files_names[i] != NULL; i++)
+   {
+      char* temp = realloc(restore_last_files_names[i], strlen(restore_last_files_names[i]) + strlen(from) + 1);
```

Comment:
`free(temp)` ?

Reply:
I guess it's not needed because I'm using `realloc`, and then assigning `temp` to `restore_last_files_names[i]`, and at the end of the function, I'm freeing `restore_last_files_names[i]`. If I understand correctly, the allocated memory for `temp` would be freed at that point, am I right? I double-checked with valgrind and it's not showing anything related to this.